From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yuki Sireneva <sylfn@disroot.org>
Date: Fri, 12 Jun 2026 10:38:03 +0000
Subject: [PATCH] (Fix) Disable TooManyCooks on ARMv7


diff --git a/Telegram/lib_crl/crl/common/crl_common_config.h b/Telegram/lib_crl/crl/common/crl_common_config.h
index 2abbcfd2b9..156bef3b9f 100644
--- a/Telegram/lib_crl/crl/common/crl_common_config.h
+++ b/Telegram/lib_crl/crl/common/crl_common_config.h
@@ -44,7 +44,7 @@
 
 #define CRL_USE_DISPATCH
 
-#elif __has_include(<tmc/ex_cpu.hpp>) && !defined CRL_FORCE_QT // dispatch && !CRL_FORCE_QT
+#elif __has_include(<tmc/ex_cpu.hpp>) && !defined __arm__ && !defined CRL_FORCE_QT // dispatch && !CRL_FORCE_QT
 
 #define CRL_USE_TMC
 
